javascript/javascript1/week3/script2.js

```diff
@@ -0,0 +1,81 @@
+//Exe- 4 Step 3: Smart-ease - Back to the basics!
+//NOnoN0nOYes (Note taking app)
+
+// Save a note
+
+//Create a variable called notes
+let notes = [];
+function saveNote(content, id) {
+    return notes.push({ content, id });
+
+}
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
+
+// Get a note
+
+function getNote(id) {
+    // your code here
+    if (typeof id !== "number" || typeof id === null || id > notes.length) {
+        alert("It's an error string");
+    } else {
+        for (let i = 0; i < notes.length; i++) {
+            if (id == notes[i].id) {
+                return notes[i];
+            }
+        }
+    }
+}
+let firstNote = getNote(1);
+console.log(firstNote); // {content: 'Pick up groceries', id: 1}
+
+//Log out notes
+function logOutNotesFormatted() {
+    for (let i = 0; i < notes.length; i++) {
+        noteInFormat = console.log("The note with id: " + notes[i].id + " has the following note text: " + notes[i].content);
```

Comment:
You can use something like `the note with id ${notes[i].id}`

Reply:
Yes right i will do that